admin/src/components/header/invite.js

```diff
@@ -173,6 +198,65 @@ const ChooseRegion = ({ value, onChange }) => {
   );
 };
 
+const ChooseCenter = ({ value, onChange, centers, onSelect }) => {
+  const { user } = useSelector((state) => state.Auth);
+
+  // useEffect(() => {
+  //   if (user.role === REFERENT_ROLES.HEAD_CENTER) {
+  //     return (
+  //       onChange({ target: { value: user.cohesionCenterId, name: "cohesionCenterId" } }),
+  //       onChange({ target: { value: user.cohesionCenterName, name: "cohesionCenterName" } })
+  //     )
+  //   }
+  //   return (
+  //     (e) => {
+  //       console.log('VALUE', JSON.stringify(e));
+  //       onChange({ target: { value: e._id, name: "cohesionCenterId" } });
+  //       onChange({ target: { value: e.value, name: "cohesionCenterName" } });
+  //       onSelect?.(e);
+  //     }
+  //   )
+  // }, []);
+
+  return (
+    // <Input
+    //   disabled={user.role === REFERENT_ROLES.HEAD_CENTER}
+    //   type="select" name="cohesionCenterName"
+    //   value={value}
+    //   onChange={(e) => {
+    //     const name = e.target.value
+    //     const _id = e.target.key
+    //     console.log('--VALUE--', name, _id, e);
+    //     onChange({ target: { value: name, name: "cohesionCenterId" } });
+    //     onChange({ target: { value: _id, name: "cohesionCenterName" } });
+    //   }}
+    // >
+    //   {centers?.map((e) => {
+    //     return (
+    //       <option value={e.name} key={e._id}>
+    //         {e.name}
+    //       </option>
+    //     );
+    //   })}
+    // </Input>
+    <ReactSelect
+      options={centers}
+      placeholder="Choisir un centre"
+      noOptionsMessage={() => "Aucun centre ne correspond à cette recherche."}
+      onChange={(e) => {
+        console.log('VALUE', JSON.stringify(e));
+        if (user.role === REFERENT_ROLES.HEAD_CENTER) {
+          onChange({ target: { value: user.cohesionCenterId, name: "cohesionCenterId" } });
+          onChange({ target: { value: user.cohesionCenterName, name: "cohesionCenterName" } });
+        }
+        onChange({ target: { value: e._id, name: "cohesionCenterId" } });
+        onChange({ target: { value: e.value, name: "cohesionCenterName" } });
+        onSelect?.(e);
```

Comment:
je ne sais pas ce que ca faisait mais ca devait être très bizarre au niveau de l'utilisateur.
=> Il peut choisir nimprote quel centre, et à la selection ca mettait son centre puis directement apres ca rechangeait avec le centre selectionné... étrange
```suggestion
        console.log('VALUE', JSON.stringify(e));
        onChange({ target: { value: e._id, name: "cohesionCenterId" } });
        onChange({ target: { value: e.value, name: "cohesionCenterName" } });
        onSelect?.(e);
```